File: preliminary_analysis/baseline_cpp_results/cpp_source/question-10-pass-0.cpp
// Minimum row or column swaps required to make every pair of adjacent cell of a Binary Matrix distinct. 
 #include <bits/stdc++.h> 
 using namespace std ;
 int minSwaps ( vector < vector < int > > & b ) {
 
 	int n = b.size() ;
 	int m = b[0].size() ;
 	int ans = 0 ;
 	int row[n] ;
 	int col[m] ;
 	for ( int i = 0 ; i < n ; i++ ) {
 		row[i] = 0 ;
 		col[i] = 0 ;
 	}
 	for ( int i = 0 ; i < n ; i++ ) {
 		for ( int j = 0 ; j < m ; j++ ) {
 			if ( b[i][j] == 1 ) {
 				row[i]++ ;
 				col[j]++ ;
 			}
 		}
 	}
 	for ( int i = 0 ; i < n ; i++ ) {
 		for ( int j = 0 ; j < m ; j++ ) {
 			if ( b[i][j] == 1 ) {
 				ans += max ( row[i] - 1, col[j] - 1 ) ;
 			}
 		}
 	}
 	return ans ;
 }
 
 int main ( ) {
 	int t ;
 	cin >> t ;
 	while ( t-- ) {
 		int n, m ;
 		cin >> n >> m ;
 		vector < vector < int > > b ( n, vector < int > ( m, 0 ) ) ;
 		for ( int i = 0 ; i < n ; i++ ) {
 			for ( int j = 0 ; j < m ; j++ ) {
 				cin >> b[i][j] ;
 			}
 		

File: preliminary_analysis/baseline_py_results/py_source/question-10-pass-0.py
# Minimum row or column swaps required to make every pair of adjacent cell of a Binary Matrix distinct. 
def minSwaps(b): 
    n = len(b) 
    # Create a 2D array to store the number of swaps 
    # required to make every pair of adjacent cell 
    # distinct. 
    dp = [[0 for i in range(n)] for j in range(n)] 
  
    # Fill the table using the above recursive formula 
    for gap in range(1, n): 
        for i in range(n - gap): 
            j = i + gap 
            if b[i] == b[j]: 
                dp[i][j] = dp[i + 1][j - 1] + 1
            else: 
                dp[i][j] = min(dp[i + 1][j], dp[i][j - 1]) + 1
  
    # Return the minimum number of swaps required 
    return dp[0][n - 1] 
  
# Driver program to test above function 
b = [1, 0, 1, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 